### PR TITLE
Add a man page for the repair command

### PR DESCRIPTION
**Pull request by mwleeds**

### CODE REVIEW

File: doc/flatpak-repair.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                <listitem><para>
+                    Show help options and exit.
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--user</option></term>
+
+                <listitem><para>
+                    Configure per-user installation.
```
Comment (matthiasclasen) [inline doc/flatpak-repair.xml]: "Repair" instead of "Configure" ?

File: doc/flatpak-repair.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--system</option></term>
+
+                <listitem><para>
+                    Configure system-wide installation.
```
Comment (matthiasclasen) [inline doc/flatpak-repair.xml]: Same here

File: doc/flatpak-repair.xml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+                </para></listitem>
+            </varlistentry>
+
+            <varlistentry>
+                <term><option>--installation=NAME</option></term>
+
+                <listitem><para>
+                    Configure the system-wide installation
```
Comment (matthiasclasen) [inline doc/flatpak-repair.xml]: And here

File: doc/flatpak-repair.xml (hunk 1 above)
Comment (mwleeds) [reply doc/flatpak-repair.xml]: Fixed

**Comment by matthiasclasen** [comment]
Looks good to me now

**Pull request merged**